Sources/TuistKit/Generator/WorkspaceGenerator.swift

```diff
@@ -59,13 +59,22 @@ final class WorkspaceGenerator: WorkspaceGenerating {
         let workspacePath = workspaceRootPath.appending(component: workspaceName)
         let workspaceData = XCWorkspaceData(children: [])
         let workspace = XCWorkspace(data: workspaceData)
+        
+        let configurationList: ConfigurationList
+        
+        if let projectConfigurations = graph.rootProject.settings?.configurations {
+            configurationList = ConfigurationList(projectConfigurations)
+        } else {
+            configurationList = ConfigurationList(BuildConfiguration.allCases.map{ Configuration(name: $0.xcodeValue, buildConfiguration: $0) })
```

Comment:
I made an assumption here about the default configurations. Need to ensure we are happy with using this enum.

Reply:
See: https://github.com/tuist/tuist/pull/238/files#diff-e7c6d2e3b554982d3152acca6998ce92R3